projects/core/src/routing/services/url-matcher.service.ts

```diff
@@ -52,26 +50,23 @@ export class UrlMatcherFactoryService {
       }
       return null;
     };
+    if (isDevMode()) {
+      matcher['matchers'] = matchers; // property added for easier debugging of routes
```

Comment:
This might be confusing for devs who could assume this is available in production. If you insist on keeping this, you might want to prefix is with an underscore?

Reply:
Added `_` prefix